src/server/game/Movement/MovementGenerators/FleeingMovementGenerator.cpp

```diff
@@ -38,7 +38,7 @@ void FleeingMovementGenerator<T>::_setTargetLocation(T* owner)
     owner->AddUnitState(UNIT_STATE_FLEEING_MOVE);
```

Comment:
37 line
```
    //Add LOS check for target point
    bool isInLOS = VMAP::VMapFactory::createOrGetVMapManager()->isInLineOfSight(owner->GetMapId(), owner->GetPositionX(), owner->GetPositionY(), owner->GetPositionZ() + 2.0f, x, y, z + 2.0f);
    if (!isInLOS)
    {
        i_nextCheckTime.Reset(500);
        return;
    }
```
This will fix the error